src/main/java/hudson/plugins/performance/build/PerformanceTestBuild.java

```diff
@@ -90,19 +92,28 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
         PrintStream logger = listener.getLogger();
         EnvVars envVars = run.getEnvironment(listener);
 
+        FilePath buildStepWorkspace = getBuildStepWorkspace(workspace);
+        try {
+            buildStepWorkspace.mkdirs();
+        } catch (IOException ex) {
+            logger.println(ex.getMessage() + " :  Permission denied");
```

Comment:
Not necessarily permission denied. Right way is to say "Cannot create directory <dir> because of error: <exception message>"